deploy/roles/r-libs/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Install packages required by R packages
+  become: yes
+  become_method: sudo
+  yum:
+    name: "{{ item }}"
+    state: present
+    disable_gpg_check: yes
+  with_items: "{{ r_dependencies }}"
+  tags:
+    - r-update
+  when: install_r_updates
```

Comment:
@lel99999 Remove `when: install_r_updates`, as that is no longer needed.  We can just run `--exclue r-libs` now to exclude the whole thing by tag.